cpu/pic32/lib/pic32_i2c.c

```diff
@@ -53,8 +53,32 @@
 #include <p32xxxx.h>
 #include <pic32_clock.h>
 #include <pic32_i2c.h>
+#include "lpm.h"
 /*----------------------------------------------------------------------------------------------*/
 #define I2C_PORT(XX)                                                     \
+  static uint32_t pic32_i2c##XX##_state = 0;                             \
+  int8_t                                                                 \
+  pic32_i2c##XX##_power_down(void)                                       \
+  {                                                                      \
+    /* Save copy of register I2C##XX##CON to restore its state           \
+     * while powering up the peripheral.                                 \
+     */                                                                  \
+    pic32_i2c##XX##_state = I2C##XX##CON;                                \
+    I2C##XX##CONCLR = _I2C##XX##CON_ON_MASK;                             \
+    PMD5SET = _PMD5_I2C##XX##MD_MASK;                                    \
+    return 0;                                                            \
+  }                                                                      \
+  int8_t                                                                 \
```

Comment:
No, I think it should still return ```int8_t``` to avoid changing the API in case we need to handle an error.